front-end/src/app/candidate/data-candidate/data-candidate.component.ts

```diff
@@ -185,11 +184,51 @@ export class DataCandidateComponent implements OnInit {
 
   // load user profile picture
   imageConstruction(){
-    if ( this.dataCandidateForm.get('profileImage').value != null ) {
+    if ( this.candidate.profileImage != null ) {
       let basePath: string = "/api/v1/uploads/";
-      this.imageUrl = environment.apiUrl + basePath + this.dataCandidateForm.get('profileImage').value;
+      this.imageUrl = environment.apiUrl + this.candidate.profileImage;
       this.nameButton = 'Change Photo';
       this.activeRemove = true;
     }
   }
-}
+
+  uploadFile(event: any){
+    let valueId: string = event.target.id;
+    
+    if ( valueId == "resumeUrl"){
+      let textResume = document.getElementById('text-' + valueId);
+      textResume.innerHTML = event.target.files[0].name;
+      this.listfile[1] = event.target.files[0];
+      this.activeResume = false;
+    }
+    else if ( valueId == "fileUrl"){
+      let textfile = document.getElementById('text-' + valueId);
+      textfile.innerHTML = event.target.files[0].name;
+      this.listfile[2] = event.target.files[0];
+      this.activeFile = false;
+    }
+  }
+
+  /*Method for loading our inputs into the candidate's edition.*/
+  editFile(){
+    if ( this.candidate.resumeUrl ){
+      document.getElementById('text-resumeUrl').innerHTML = this.candidate.resumeUrl.slice(25);
```

Comment:
Seems this code is handling URL and a specific one like `http://localhost:8080`. Can you check if this will work for longer domains like `http://ec2-54-149-160-98.us-west-2.compute.amazonaws.com`?

Reply:
> Seems this code is handling URL and a specific one like `http://localhost:8080`. Can you check if this will work for longer domains like `http://ec2-54-149-160-98.us-west-2.compute.amazonaws.com`?

The candidate's resume attribute is saved with the file name followed by the calendar and time that file was uploaded. Opt for this method so as not to generate a conflict in case files with the same name are uploaded, the idea is to generate a single name. 

Example: 
 `file-year-month-day-time-minute-indicator(am or pm)-namefile.pdf`

I just extract the name there for display in the front end. Clicking on the download icon builds the url of the file. The code you mentioned does not bring the whole domain, only the name of the file with which it was saved in the database.

Reply:
I get it. Makes sense. I'll test the functionality in minutes.